draft-ietf-lamps-csr-attestation.md

```diff
@@ -527,34 +527,45 @@ EvidenceStatement ::= SEQUENCE {
-the Relying Party to perform any parsing of the data in `EvidenceStatement.stmt`.
-The type OID, which identifies the format of the data found in the Evidence statement,
-typically suffices for a Relying Party to select the correct
-Verifier (software) to invoke. However, in some cases the Relying Party
-may have more than one Verifier capable of parsing a given type OID -- for
-example if the OID indicates a wrapper format such as DICE
-ConceptualMessageWrapper that can contain further proprietary data.
-A design goal of this specification is to enable Relying Parties to
-select an appropriate Verifier (software) without the need to perform any
-parsing of the `EvidenceStatement.stmt` data.
-To help with this, the Attester MAY populate the hint with a name of a
-software package that will be capable of parsing this data.
-The hint SHOULD contain a value that is unique
-to this Verifier, for example,  a fully qualified domain name (FQDN), a uniform
-resource name (URN) {{RFC8141}}, or a registered value corresponding to this
-Evidence format.
-In a typical usage scenario, the RP is pre-configured with a list of trusted
-Verifiers and the corresponding hint values can be used to look up appropriate Verifier.
-Tricking an RP into interacting with an unknown and untrusted Verifier has to be avoided
-as the retrieved Attestation Result must be reliable.
+Based on the responsibilities of the different roles in the RATS architecture,
+Relying Parties need to relay Evidence to Verifiers for evaluation and obtain
+an Attestation Result in return. Ideally, the Relying Party should select a Verifier
+based on the received Evidence without requiring the Relying Party to inspect the
+Evidence itself. This "routing" decision is simple when there is only a single
+Verifier configured for use by a Relying Party but gets more complex when there
+are different Verifiers available and each of them capable of parsing only certain
+Evidence formats.
+
+In some cases, the EvidenceStatement.type OID will be sufficient information
+for the Relying Party to correctly route it to an appropriate Verifier,
+however since the type OID only identifies the general data format, it is possible
+that multiple Verifiers are registered against the same type OID in which case the
+Relying Party will either require additional parsing of the evidence statement, or
+the Attester will be required to provide additional information.
+
+To simplify the task for the Relying Party an optional field, the hint, is available
+in the EvidenceStatement structure, as shown in {{code-EvidenceStatement}}. An
+Attester MAY include the hint to the EvidenceStatement and it MAY be processed
+by the Relying Party. The Relying Party MAY decide not to trust the information
+embedded in the hint or policy MAY override any information provided by the Attester
+via this hint.
+
+When the Attester populates the hint, it MUST contain a fully qualified domain
+name (FQDN) which uniquely identifies a Verifier.
```

Comment:
Non-blocking recommendation: perhaps the creation of an informative reference to [RFC6125](https://datatracker.ietf.org/doc/html/rfc6125#section-2.2) and relevant implications? It seems RFC1035 is too high-level and only refers to the traditional terms so may not be so helpful. If you do that and cite it here, it is something I would want to look up to know what "exactly you mean by FQDN" like I did during this review.

Reply:
Nit - RFC6125 was obsoleted by RFC9525. It seems odd to reference this RFC (on TLS service identity) for a value that is not intended to be de-referenced. If it may be the case that one day it'd be desirable to dereference verifier hints, this field should be extensible with the current non-dereferenceable name notion the first (and initially only) instantiation.

Reply:
> Nit - RFC6125 was obsoleted by RFC9525. It seems odd to reference this RFC (on TLS service identity) for a value that is not intended to be de-referenced.

I do not disagree, I could not really find the right reference to explain what FQDN is or is not without other necessary context, but I thought per the title, "Representation and Verification of Domain-Based Application Service
 Identity within Internet Public Key Infrastructure Using X.509," aside from the TLS identity context. Are you suggesting a reference to RFC1034 is more suitable? Ironically 1034 does not reference the term FQDN, it seems to evolved from fully qualified address, total, or absolute domain. FQDN is not defined, so I thought it would be counterproductive.

If RFC1034 is not helpful, is there another you would recommend?